airbyte-to-flow/src/interceptors/airbyte_source_interceptor.rs

```diff
@@ -366,7 +370,18 @@ impl AirbyteSourceInterceptor {
                         for (i, binding) in c.bindings.iter().enumerate() {
                             let resource: ResourceSpec =
                                 serde_json::from_str(&binding.resource_spec_json)?;
-                            stb.lock().await.insert(resource.stream.clone(), i);
+
+                            let normalizations = std::fs::read_to_string(format!(
+                                "{}/{}{}",
+                                STREAM_PATCH_DIR_NAME,
+                                &resource.stream,
```

Comment:
At this point, I'm pretty sure `resource.stream` is directly usable without needing to do `stream_to_recommended_name`. Let me know if I'm wrong about that.